src/coverlet.console/Program.cs

```diff
@@ -174,26 +175,60 @@ static int Main(string[] args)
                 }
 
                 var thresholdTypeFlags = ThresholdTypeFlags.None;
```

Comment:
Sorry but I have one last request. Now that ```thresholdTypeFlags``` isn't passed to ```GetThresholdTypesBelowThreshold``` anymore. The variable here and in the following ```foreach``` can be removed.
The definition for ```thresholdTypeFlags``` can then be made in line 268 like 
```csharp
var thresholdTypeFlags = result.GetThresholdTypesBelowThreshold(summary, thresholdTypeFlagValues, dThresholdStat);
```

The same should be done in ```CoverageResultTask```.

Reply:
removed in both places. please check @daveMueller

Reply:
Looks good. Thanks.